src/actions/actionActions.ts

```diff
@@ -70,7 +70,7 @@ export const editActionThunkAsync = (appId: string, action: ActionBase) => {
 
             // Fetch train dialogs if any train dialogs were impacted
             if (deleteEditResponse.trainDialogIds && deleteEditResponse.trainDialogIds.length > 0) {
-                dispatch(fetchAllTrainDialogsThunkAsync(appId))
+                await dispatch(fetchAllTrainDialogsThunkAsync(appId))
```

Comment:
I don't think this is what we want. Most of these `dispatch` calls are supposed to fire-forget style / non-awaited.

Reply:
I believe if you put a "void" in front of that call it tells tslint that you intended not to await the promise so:

`void dispatch(fetchAllTrainDialogsThunkAsync(appId))`

Reply:
Lars' suggestion seems to mollify the linter!